packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -298,9 +300,9 @@ class IAPError {
   /// The error code.
   final String code;
 
-  /// A human-readable error message, possibly null.
+  /// A human-readable error message.
```

Comment:
Why the change here?

Reply:
I think empty error message and null error message means the same thing, so I thought we could make this non null so we don't have 2 values mean the same thing.